readme.md

```diff
@@ -1,26 +1,2 @@
-# Style it up
-
-Your task is to change `style.css` in order to get the page in `index.html` 
-looking and behaving just as described below.
-
-Please **do not** modify or replace the HTML itself. Do take a look at what’s 
-inside `index.html` though.
-
-The page consists of a header, a footer, and 9 blocks between them.
-
-The width of the blocks must be 300 pixels. The height of each of the blocks 
-(including both header and footer) has to be 100 pixels. When hovered (the 
-cursor is on the block), height must increase to 200 pixels.
-The color of the header must be blue, the color of the footer must be black, and 
-the colors of the blocks in between must be the following
-1. red
-1. green
-1. orange
-
-There should be no vertical spacing between the blocks and the edges of the 
-page. Place the blocks so that they are horizontally centered on the page.
-
----
-**Read the guideline before start**
-
-[Guideline](https://github.com/mate-academy/layout_task-guideline/blob/master/README.md)
+https://ivandeiko.github.io/layout_style-it-up/ - demo link
+https://ivandeiko.github.io/layout_style-it-up/report/html_report/ - test report
```

Comment:
why have you removed all description? Please bring it back and add demo links with markdown styling.
`[Demo Link](https://ivandeiko.github.io/layout_style-it-up/)`